ShopperInsights/src/test/java/com/braintreepayments/api/EligiblePaymentsApiUnitTest.kt

```diff
@@ -0,0 +1,123 @@
+package com.braintreepayments.api
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+
+class EligiblePaymentsApiUnitTest {
+
+    private lateinit var sut: EligiblePaymentsApi
+    private lateinit var callback: EligiblePaymentsCallback
+    private lateinit var braintreeClient: BraintreeClient
+    private val configuration: Configuration = createMockConfiguration()
+
+    @Before
+    fun setup() {
+        callback = mockk(relaxed = true)
+        braintreeClient = mockk(relaxed = true)
+        setupBraintreeClientToReturnConfiguration()
+        sut = EligiblePaymentsApi(braintreeClient)
+    }
+
+    @Test
+    fun test_sendPOST_Error() {
```

Comment:
Nit - in Kotlin we can use backticks to include spaces in the test function names.


```suggestion
    fun `when sendPost is called and an error occurs, callback onResult is invoked with the error`() {
```